invenio_rdm_records/resources/serializers/bibtex/__init__.py

```diff
@@ -22,6 +25,7 @@ def __init__(self, **options):
             format_serializer_cls=SimpleSerializer,
             object_schema_cls=BibTexSchema,
             list_schema_cls=BaseListSchema,
+            schema_kwargs={"dumpers": bibtex_dumpers},
```

Comment:
note: the mutable values here are fine, because they're used as arguments for _calling_ a function; the issue is only when mutable objects are being used as the default values for function parameters (i.e. when defining a function)